models/articles.model.js

```diff
@@ -80,3 +80,28 @@ exports.selectCommentsByArticleId = (id) => {
     })
     .then(({ rows }) => rows);
 };
+
+exports.insertCommentForArticleId = (id, comment) => {
+  const commentFormat = [
+    ["body", "string"],
+    ["username", "string"],
+  ];
+  if (!Object.keys(comment).length) {
+    return Promise.reject({ code: 400, msg: "Bad request" });
+  }
+  if (!commentFormat.every(([key, type]) => typeof comment[key] === type)) {
+    return Promise.reject({ code: 422, msg: "Unprocessable entity" });
+  }
+  return db
+    .query(
+      `
+      INSERT INTO
+        comments (body, author, votes, created_at, article_id)
+      VALUES
+        ($1, $2, $3, $4, $5)
+      RETURNING *;
+      `,
```

Comment:
As mentioned in your tests, you don't need to handle 422 errors. Please see my comments 😄

Reply:
So I should get rid of any paths that lead to 422 and they should instead 400 errors?